tensorflow/compiler/mlir/tosa/transforms/legalize_tfl.cc

```diff
@@ -2374,6 +2376,49 @@ LogicalResult ConvertTFLSoftmaxOp::matchAndRewrite(
   return success();
 }
 
+LogicalResult ConvertTFLRsqrtOp::matchAndRewrite(
+    Operation* op, PatternRewriter& rewriter) const {
+  auto tfl_rsqrt_op = cast<TFL::RsqrtOp>(op);
+
+  RankedTensorType output_type =
+      tfl_rsqrt_op.getResult().getType().dyn_cast<RankedTensorType>();
+  RankedTensorType input_type =
+      tfl_rsqrt_op.getX().getType().dyn_cast<RankedTensorType>();
+
+  // Quantization case
+  if (input_type.getElementType().isa<mlir::quant::QuantizedType>() &&
+      output_type.getElementType().isa<mlir::quant::QuantizedType>()) {
+    mlir::quant::UniformQuantizedType input_qtype =
```

Comment:
I agree with this. However, when I moved to the outside of the if statement, it's causing some other irrelevant Lit test case failures which I have no idea why. 

Since this is not the only case, I would suggest keep as it is and do another patch to refactor all the instances and fix relevant lit failures.